modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -248,19 +263,12 @@ public function widget( $args, $instance ) {
 	private function maybe_render_sort_javascript( $instance, $order, $orderby ) {
 		if ( ! empty( $instance['user_sort_enabled'] ) ) :
 		?>
-		<!--
-		This JS is a bit complicated, but here's what it's trying to do:
-		- find or create a search form
-		- find or create the orderby/order fields with default values
-		- detect changes to the sort field, if it exists, and use it to set the order field values
-		-->
 		<script type="text/javascript">
 				jQuery( document ).ready( function( $ ) {
-					var actionUrl      = <?php echo json_encode( home_url( '/' ) ); ?>,
-						orderByDefault = <?php echo json_encode( $orderby ); ?>,
+					var orderByDefault = <?php echo json_encode( $orderby ); ?>,
 						orderDefault   = <?php echo json_encode( $order ); ?>,
 						widgetId       = <?php echo json_encode( $this->id ); ?>,
-						currentSearch  = <?php echo json_encode( isset( $_GET['s'] ) ? $_GET['s'] : '' ); ?>
+						currentSearch  = <?php echo json_encode( isset( $_GET['s'] ) ? $_GET['s'] : '' ); ?>;
```

Comment:
I love where this is going. Can we go a step further maybe and move this bit of JS to the file too, so we can just use `wp_localize_script` to pass all these variables? We can check the `maybe` part inside the JS file.

Reply:
@zinigor I actually started doing exactly that but realized that the JS is unique per widget and weird things would likely happen if there were multiple copies of the widget being used at once. I decided to punt it for a future PR.